builds/Resources/Verilator_resources/verilator_config.vlt

```diff
@@ -3,10 +3,10 @@
 // Flags for verilator
 
 `verilator_config
-lint_off -msg WIDTH
-lint_off -msg CASEINCOMPLETE
-lint_off -msg STMTDLY
-lint_off -msg INITIALDLY
-lint_off -msg UNSIGNED
-lint_off -msg CMPCONST
+lint_off -rule WIDTH
+lint_off -rule CASEINCOMPLETE
+lint_off -rule STMTDLY
+lint_off -rule INITIALDLY
+lint_off -rule UNSIGNED
+lint_off -rule CMPCONST
```

Comment:
I believe this breaks versions of Verilator prior to 4.026, released on 2020-01-11 (added upstream in https://github.com/verilator/verilator/commit/b7665a88dbab6c47b26489999854d577dff78b87). @rsnikhil please revert this change, as it does not work with the version of Verilator shipped in Ubuntu 18.04, and unless I am mistaken will also not work with the version shipped in Debian 10 "buster".